packages/paste-libraries/use-clipboard/package.json

```
@@ -0,0 +1,38 @@
+{
+  "name": "@twilio-paste/use-clipboard-library",
+  "version": "0.2.1",
+  "category": "library",
+  "status": "production",
+  "description": "A library to copy text to clipboard",
+  "author": "Twilio Inc.",
+  "license": "MIT",
+  "main:dev": "src/index.tsx",
+  "main": "dist/index.js",
+  "module": "dist/index.es.js",
+  "types": "dist/index.d.ts",
+  "sideEffects": false,
+  "publishConfig": {
+    "access": "public"
+  },
+  "files": [
+    "dist"
+  ],
+  "scripts": {
+    "build": "yarn clean && NODE_ENV=production node build.js && tsc",
+    "build:js": "NODE_ENV=development node build.js",
+    "build:props": "typedoc --tsconfig ./tsconfig.json --json ./dist/prop-types.json",
+    "clean": "rm -rf ./dist",
+    "tsc": "tsc"
+  },
+  "dependencies": {
+    "use-clipboard-copy": "^0.2.0"
```

```suggestion
    "use-clipboard-copy": "0.2.0"
```

We pin library deps to avoid any funkiness.